Source/ChocolateyGui/Properties/Resources.nb.resx

```diff
@@ -0,0 +1,553 @@
+  </data>
+  <data name="SettingsView_Settings" xml:space="preserve">
+    <value>Innstillinger</value>
+  </data>
+  <data name="SettingsView_Sources" xml:space="preserve">
+    <value>Kilder</value>
+  </data>
+  <data name="SettingsView_SourcesCertificate" xml:space="preserve">
+    <value>Sertifikat</value>
+  </data>
+  <data name="SettingsView_SourcesCertificatePass" xml:space="preserve">
+    <value>Sertifikatpassord</value>
+  </data>
+  <data name="SettingsView_SourcesId" xml:space="preserve">
+    <value>Id</value>
+  </data>
+  <data name="SettingsView_SourcesPassword" xml:space="preserve">
+    <value>Passord</value>
+  </data>
+  <data name="SettingsView_SourcesPriority" xml:space="preserve">
+    <value>Prioritet</value>
+  </data>
+  <data name="SettingsView_SourcesSource" xml:space="preserve">
+    <value>Kilde</value>
+  </data>
+  <data name="SettingsView_SourcesUsername" xml:space="preserve">
+    <value>Brukernavn</value>
+  </data>
+  <data name="SettingsView_ToggleShowConsoleOutput" xml:space="preserve">
+    <value>Vis konsolltekst</value>
+  </data>
+  <data name="ShellView_ButtonAbout" xml:space="preserve">
+    <value>om</value>
+  </data>
+  <data name="ShellView_ButtonSettings" xml:space="preserve">
+    <value>innstillinger</value>
+  </data>
+  <data name="SettingsView_SourcesPath" xml:space="preserve">
+    <value>Sti</value>
+  </data>
+  <data name="LocalSourceView_CheckboxMatchExact" xml:space="preserve">
+    <value>Treff på eksakte ord</value>
+  </data>
+  <data name="RemoteSourceView_CheckboxMatchExactly" xml:space="preserve">
+    <value>Treff på eksakte ord</value>
+  </data>
+  <data name="SettingsView_ButtonRemove" xml:space="preserve">
+    <value>Fjern</value>
+  </data>
+  <data name="SettingsView_SourcesIsDisabled" xml:space="preserve">
+    <value>Er deaktivert</value>
+  </data>
+  <data name="SettingsView_SourcesDisabled" xml:space="preserve">
+    <value>Deaktivert</value>
+  </data>
+  <data name="Controls_PackagesContextMenuInstall" xml:space="preserve">
+    <value>Installer</value>
+  </data>
+  <data name="Controls_PackagesContextMenuPin" xml:space="preserve">
+    <value>Fest</value>
+  </data>
+  <data name="Controls_PackagesContextMenuReinstall" xml:space="preserve">
+    <value>Reinstaller</value>
+  </data>
+  <data name="Controls_PackagesContextMenuUninstall" xml:space="preserve">
+    <value>Avinstaller</value>
+  </data>
+  <data name="Controls_PackagesContextMenuUnpin" xml:space="preserve">
+    <value>Løsne</value>
+  </data>
+  <data name="Controls_PackagesContextMenuUpdate" xml:space="preserve">
+    <value>Oppdater</value>
+  </data>
+  <data name="Controls_PackagesContextMenuDetails" xml:space="preserve">
+    <value>Detaljer</value>
+  </data>
+  <data name="PackageView_ButtonPin" xml:space="preserve">
+    <value>Fest</value>
+  </data>
+  <data name="Bootstrapper_UnhandledException" xml:space="preserve">
+    <value>Ubehandled Unntak</value>

```

Comment:
Ja, man må vel vite kontekst for å si noe om det skal være entall eller flertall. "Unhandled" kan jo benyttes i begge tilfeller på engelsk.